pages/api/templates.ts

```diff
@@ -5,7 +5,7 @@ import templatesSchema from "@lib/middleware/schemas/templates.schema.json";
 import { NextApiRequest, NextApiResponse } from "next";
 import { isAdmin } from "@lib/auth";
 import { logAdminActivity } from "@lib/adminLogs";
-import { AdminLogAction, AdminLogEvent } from "@lib/types";
+import { AdminLogAction, AdminLogEvent } from "@lib/types/utility-types";
```

Comment:
If AdminLogAction and AdminLogEvent are enums that are needed to invoke logAdminActivity I'm wondering if we should move them back to being exported explicity from the @lib/adminLogs module.  This way only a single import is needed.